apps/studio/components/layouts/AuthLayout/AuthLayout.utils.ts

```diff
@@ -33,6 +35,26 @@ export const generateAuthMenu = (
         { name: 'Users', key: 'users', url: `/project/${ref}/auth/users`, items: [] },
       ],
     },
+    ...(authenticationShowSecurityNotifications
+      ? [
+          {
+            title: 'Notifications',
+            items: [
+              ...(authenticationEmails
+                ? [
+                    {
+                      name: 'Email',
+                      key: 'email',
+                      pages: ['templates', 'smtp'],
+                      url: `/project/${ref}/auth/templates`,
+                      items: [],
+                    },
+                  ]
+                : []),
+            ],
+          },
+        ]
+      : []),
```

Comment:
Should this be wrapped in a `IS_PLATFORM` check like some items in _Configuration_?